### DISCUSSION TITLE
why? Querying MariaDB, returned rows, but in output "No data - zero rows fetched, selected, or processed"

### DISCUSSION THREAD

**Discussion by SilenyKrecek** [dbeaver/dbeaver#19498] [Q&A] [answered]
Hello, I am a newbie to DBeaver (and to MariaDB/MySQL too). Because I did not find anything through Google or otherwise, I am asking here.

DBeaver 23.0.1 - MariaDB (5.5.x - don't ask). 

When I write simple select, which returns rows (the table has some 300+ rows), I get a message "No data - zero rows fetched, selected, or processed". Also when listing through returned rows in the grid (fetch size 50) I get the message on each next fetch.

Any hints on what causes this and how to get rid of it? 

Thanks!

**Comment by LonwoLonwo** [comment]
Could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Reply by SilenyKrecek** [reply]
Hello, while I was trying to produce as small logs as possible, I noticed, that this is not general behaviour. These messages are produced by select operations only on some objects.
I started using DBeaver on database already in use for some time (before I started using DBeaver) so I did with DBeaver select almost exclusively on these objects, so I thought it was a general thing against that particular MariaDB database (I use it for Postgres on other "projects" where it worked fine).
Please allow me a few days to investigate more, I'll post results (with logs if not nailed cause) there.

**Reply by SilenyKrecek** [reply, marked as answer]
Hello LonwoLonwo,

I found the cause, and it is not a DBeaver issue. In almost all selects I ran since the start of using DBeaver a function was used (for assembling full name from first name, last name and academic degrees etc.) which generates a "no data..." message, when used on null input (PersonID) - for example on days, where no one is scheduled for a particular task.
I did not notice it in db tool I had used before, as there these messages were in place which was not shown by default.
So, it is now solved in regard to DBeaver, now I have to find out if these messages can be turned off somehow in the context of MariaDB.

Happy Easter (if You celebrate it)!
